### DISCUSSION TITLE
Is there a bug with THAnull ?

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#71] [Ideas]
Il est nul ce THA !
Tu ne trouves pas, @AlZorba ?
Donc on a des url avec THANull : https://thacer.archaiodata.com/ceram.php?ID=8017&THAnull&INV=null
Je me demande si ce qui est voulu n'est pas ANA=THAnull ? (en fait ANA=THA<un nombre>)

**Comment by AlZorba** [comment]
Remédié avec l'ID unique
